salt/unity/init.sls

```diff
@@ -0,0 +1,11 @@
+unity:
+    file.managed:
+        - name: /opt/Unity-{{ grains.version }}.pkg
```

Comment:
The version and hash should be specified in pillar like we do for other versioned stuff, ie something like
```yaml
{% set version_info = pillar.get('unity', {}).get('version_info', '2017.2 sha256=deadbeef') %}
{% set version, version_hash = version_info.split() %}
```